test/iht/controllers/application/assets/insurancePolicy/InsurancePolicyOverviewControllerTest.scala

```diff
@@ -76,9 +76,9 @@ class InsurancePolicyOverviewControllerTest extends ApplicationControllerTest {
       status(result) shouldBe OK
 
```

Comment:
The text "Did John Smith(M/CP) ever pay insurance premiums for a policy that wasn’t due to pay out to their estate?" should read "Did John Smith(M/CP) ever pay insurance premiums for a policy that **was not** due to pay out to their estate?". This is on both overview and question page.